src/Components/Components/src/Routing/Router.cs

```diff
@@ -55,6 +55,7 @@ public class Router : IComponent, IHandleAfterRender, IDisposable
         [Parameter] public RenderFragment AuthorizingContent { get; private set; }
 
         private RouteTable Routes { get; set; }
+        private bool RouteTableInitialized { get; set; }
```

Comment:
Looks like this could be dropped if `RouteTable` was a reference type.

If not dropping it though, as a minor nitpick, consider converting it to a field.